data_structures/queue/queue_on_two_stacks.py

```diff
@@ -0,0 +1,123 @@
+"""Queue implementation using two stacks"""
+
+from typing import Any
+
+
+class Queue:
```

Comment:
Please name the class something else so it does not conflict with `queue.Queue`

Reply:
@cclauss  Sure. Do you have any naming convention in mind so that I make it consistent with other queue implementations like [this PR](https://github.com/TheAlgorithms/Python/pull/8608)? Which one is more suitable? `Queue_TwoStacks`, `QueueByStacks` or `QueueTwoStacks`.

Reply:
No underscore in ClassNames --> https://beta.ruff.rs/docs/rules/#pep8-naming-n

You can run `ruff --select=N path/to/my/python/file.py` to ensure all is OK.